colabs/wandb-model-registry/New_Model_Logging_in_W&B.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #148.    num_samples = 100
</pre><p>Seperate cell for classes vs. global defined variables/code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>